build-plugin/src/main/kotlin/ThunderbirdProjectConfig.kt

```diff
@@ -0,0 +1,18 @@
+import org.gradle.api.JavaVersion
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+object ThunderbirdProjectConfig {
+
+    object Android {
+        const val sdkMin = 21
+
+        // Only needed for application
+        const val sdkTarget = 35
+        const val sdkCompile = 35
```

Comment:
As this is only required for the application, it wouldn't make sense to wrap it under an object called `Application`?
```suggestion
        object Application {
            const val sdkTarget = 35
            const val sdkCompile = 35
        }
```